2-collectors/scala-stream-collector/src/main/scala/com.snowplowanalytics.snowplow.collectors/scalastream/sinks/KafkaSink.scala

```diff
@@ -57,11 +57,12 @@ class KafkaSink(config: CollectorConfig, inputType: InputType.InputType) extends
     props.put("bootstrap.servers", config.kafkaBrokers)
     props.put("acks", "all")
     props.put("retries", "0")
-    props.put("batch.size", config.byteLimit.toString)
+    props.put("buffer.memory", config.byteLimit.toString)
+    props.put("batch.size", config.recordLimit.toString)
```

Comment:
mmmh, the `record-limit` configuration represents a number of records, `batch.size` is a number of bytes, I don't think it really applies here.